example/scripts/emulators/pocketbase.sh

```
@@ -0,0 +1,114 @@
+}
+
+setup_schema() {
+  local project_dir="$1"
+  cd "$project_dir" || return 1
+  
+  # Create superuser
+  pocketbase superuser upsert test@test.com testpassword123 --dir="pb_data" > /dev/null 2>&1
+  sleep 2
+  
+  # Get admin token
+  local admin_token=$(curl -s -X POST http://127.0.0.1:8090/api/collections/_superusers/auth-with-password \
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)